entrypoint.sh

```diff
@@ -72,6 +72,9 @@ process_config() {
     if [[ -n $CHECK_NEW_LISTING_ENABLED ]]; then
         sed -i "s/checkNewListingEnabled.*/checkNewListingEnabled: ${CHECK_NEW_LISTING_ENABLED}/" config.yml
     fi
+    if [[ -n $TPD_NEAREST_HOUR ]]; then
+        sed -i "s/tpdNearestHour.*/tpdNearestHour: ${TPD_NEAREST_HOUR}/" config.yml
+    fi
```

Comment:
@patbaumgartner Is this the proper method to add the parameter to the entrypoint file for docker?

I followed the naming convention for the other parameters above, but I'm not sure if it's correct and whether changes in other areas will be needed.